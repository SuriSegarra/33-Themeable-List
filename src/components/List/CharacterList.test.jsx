import React from 'react';
import { shallow } from 'enzyme';
import CharactersList from './CharacterList';

const characters = [
  {
    id:'2',
    image: 'someimage.jpeg',
    name: 'Rick Sanchez',
    species:'human',
    status: 'alive',
    origin: 'earth'
  }
];

describe('Characters', () => {
  it('renders Characters List', () => {
    const wrapper = shallow(<CharactersList characters={characters}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
