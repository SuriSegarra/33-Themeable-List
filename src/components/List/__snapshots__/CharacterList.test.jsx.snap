// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Characters renders Characters List 1`] = `
<ul>
  <li
    key="2"
  >
    <Character
      id="2"
      image="someimage.jpeg"
      name="Rick Sanchez"
      origin="earth"
      species="human"
      status="alive"
    />
  </li>
</ul>
`;
